pandas/io/parsers.py

```diff
@@ -999,6 +999,8 @@ def _failover_to_python(self):
 
     def read(self, nrows=None):
         if nrows is not None:
+            nrows = _validate_integer('nrows', nrows)
```

Comment:
I think this is unnecessary.  This already gets validated the moment you call `read_excel`.